### PR TITLE
First iteration of REST API

### PR DESCRIPTION
**Pull request by duizer**

### CODE REVIEW

File: src/ProjectOrigin.WalletSystem.IntegrationTests/QueryCertificatesTest.cs [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
-using FluentAssertions;
-using Grpc.Core;
-using ProjectOrigin.WalletSystem.Server.Extensions;
-using ProjectOrigin.WalletSystem.V1;
 using Xunit;
-using WalletService = ProjectOrigin.WalletSystem.V1.WalletService;
 using Xunit.Abstractions;
 using GranularCertificateType = ProjectOrigin.WalletSystem.Server.Models.GranularCertificateType;
-using ProjectOrigin.HierarchicalDeterministicKeys.Interfaces;
+using WalletService = ProjectOrigin.WalletSystem.V1.WalletService;
```
Comment (MartinSchmidt) [inline src/ProjectOrigin.WalletSystem.IntegrationTests/QueryCertificatesTest.cs:19]: why these changes?

File: src/ProjectOrigin.WalletSystem.Server/Models/CertificateViewModel.cs [hunk 2]
```diff
@@ -1,15 +1,14 @@
...
 
 namespace ProjectOrigin.WalletSystem.Server.Models;
 
 public record CertificateViewModel
 {
     public required Guid Id { get; init; }
-    public required string Registry { get; init; } = string.Empty;
+    public required string RegistryName { get; init; } = string.Empty;
     public required GranularCertificateType CertificateType { get; init; }
     public required string GridArea { get; init; } = string.Empty;
```
Comment (MartinSchmidt) [inline src/ProjectOrigin.WalletSystem.Server/Models/CertificateViewModel.cs:13]: string.Empty should probably removed, don't even know why they are there, will you are changing this.

File: src/ProjectOrigin.WalletSystem.Server/Services/GRPC/RegistryService.cs [hunk 3]
```diff
@@ -6,7 +6,7 @@
 using ProjectOrigin.WalletSystem.Server.Options;
 using ProjectOrigin.WalletSystem.Server.Projections;
 
-namespace ProjectOrigin.WalletSystem.Server.Services;
+namespace ProjectOrigin.WalletSystem.Server.Services.GRPC;
```
Comment (MartinSchmidt) [inline src/ProjectOrigin.WalletSystem.Server/Services/GRPC/RegistryService.cs:9]: This is not an actual gRPC service, more an internal service that helps contact the registry, should probably be moved back to services... or elsewhere.

**Review by MartinSchmidt — CHANGES_REQUESTED** [review]

File: src/ProjectOrigin.WalletSystem.IntegrationTests/QueryCertificatesTest.cs (hunk 1 above)
Comment (duizer) [reply src/ProjectOrigin.WalletSystem.IntegrationTests/QueryCertificatesTest.cs:19]: I have a habit of pressing Ctrl+R+G when I edit a file. This invokes ReSharper to sort the the using statements and remove obsolete using statements. 

It is a leftover from when I started writing the tests. I can remove this if you want.

**Review by MartinSchmidt — APPROVED** [review]
